.github/workflows/toranj.yml

```diff
@@ -180,9 +180,6 @@ jobs:
         ./tests/toranj/build.sh --log-level CRIT all
         git clean -dfx
         ./tests/toranj/build.sh --log-level NONE all
-        #- - - - - - - - - - - - - - - - - - - - - - - - - - -
-        git clean -dfx
-        ./tests/toranj/build.sh --enable-plat-key-ref all
```

Comment:
I see you removed this.

I added this back in 
- #9620.

From what I recall, the issue was that after the KEY_REF code was contributed to OT, there was no GitHub CI test covering this, or at a minimum, checking whether the OT stack would compile correctly (with no errors) when this feature is enabled. This is important because as new PRs are merged, we want to ensure we at least build with this feature enabled so that no new PR breaks the build.

If you are removing this, I assume we are now going to cover this feature in one of the newly added CI jobs? Just asking here to confirm. Thanks. 😃